echopype/calibrate/calibrate_azfp.py

```diff
@@ -154,6 +154,9 @@ def _cal_power(self, cal_type, **kwargs):
         # Add env and cal parameters
         out = self._add_params_to_output(out)
 
+        # Order the dimensions
+        out["echo_range"] = out["echo_range"].transpose("channel", "ping_time", "range_sample")
+
```

Comment:
@leewujung I am wondering if this is the most appropriately place for this. This line is necessary because of how we create the EchoData object for AZFP, correct? If so, should we instead correct the order in `set_groups_azfp`? I am fine with leaving this as is, I just thought I would bring this up.

Reply:
Good point, I agree this is the best done in `set_groups_azfp`. In fact this was part of my test to see if we can do without `.drop("channel")`  in this line:
https://github.com/OSOceanAcoustics/echopype/blob/b6205e57a13fea2e50801f099ea4cea4c38855da/echopype/preprocess/api.py#L79

Somehow that extra dimension/coordinate only shows up for AZFP data, and I was wondering if it had anything to do with the order of the dimension, but that wasn't the reason.

I'll create an issue so we remember to do this in `set_groups_azfp` instead of here.

Reply:
Sounds good!

Reply:
Oh actually, now that I looked at this again, the order of dimension to be changed is for `echo_range` and not `Sv` (which came from `backscatter_r`), so I guess nothing has to change for `set_groups_azfp`, but it is the AZFP part of `compute_range` that needs to change.

Reply:
I was having a little trouble tracking down where that happened. The AZFP part of `compute_range` was using parts of the EchoData object, so I assumed that we needed to make corrections in `set_groups_azfp`. After looking into this a little further, I found that in the `EK` else statement of `compute_range` we do `range_meter = range_meter.transpose("channel", "ping_time", "range_sample")`, so maybe the fix is as simple as adding that to the AZFP if statement block.

Reply:
Oh right, cool that you tracked it down! I'll link your comment to that issue... v0.6.2... so that we don't have to do this now. :P